develop.html.mako

```diff
@@ -27,22 +27,40 @@ on getting started.
 <a name="developers"></a>
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where do you find the developers ?  Four places:</p>
```

Comment:
haha

Reply:
These is actually also https://gitter.im/sqlalchemy/community, so I guess 5 places :P

Reply:
Maybe it would be better to divide in 3 sections:
- mailing list where both are mentioned
- gitter where both channels are mentioned
- irc where again both channells are mentioned

Reply:
This is for the [/develop.html](https://www.sqlalchemy.org/develop.html) page which is aimed at code contribution.  

There is also the [/support.html](https://www.sqlalchemy.org/support.html) page, which should contain the community room (I'll add it!)

The point on this page is to showcase the developer stuff, and politely shift people to the public community stuff for everything but library/ecosystem maintenance.

Reply:
Oh ok. So maybe the other gitter link should go in the support page?

Reply:
I do like the organizational idea though, i'm going to try shifting things again!